enterprise/causal-clustering/src/main/java/org/neo4j/causalclustering/ReplicationModule.java

```diff
@@ -78,6 +78,7 @@ public ReplicationModule( MemberId myself, PlatformModule platformModule, Config
 
         TimeoutStrategy progressRetryStrategy = new ExponentialBackoffStrategy( initialBackoff, upperBoundBackoff );
         TimeoutStrategy leaderRetryStrategy = new ConstantTimeTimeoutStrategy( leaderBackoff );
+        long availabilityTimeoutMillis = config.get( CausalClusteringSettings.replication_retry_timeout_base ).toMillis();
```

Comment:
should it not be CausalClusteringSettings.replication_retry_timeout_limit?

Reply:
That limit is for the maximum backoff. I just want a reasonable small timeout for awaiting availability. It could have been its own setting, but I think this one is in the same ballpark and it's not too important to tweak it.

Reply:
Ok. I think I misunderstood